FakeItEasy.Deploy/CommandLineOptions.cs

```diff
@@ -0,0 +1,83 @@
+namespace FakeItEasy.Deploy;
+
+internal class CommandLineOptions
+{
+    public string Repo { get; private set; } = string.Empty;
+    public string TagName { get; private set; } = string.Empty;
+    public string ArtifactsFolder { get; private set; } = string.Empty;
+
+    public static CommandLineOptions Parse(string[] args)
+    {
+        Action<string>? nextArgAction = null;
+        var options = new CommandLineOptions();
+        foreach (var arg in args)
+        {
+            if (nextArgAction is not null)
+            {
+                nextArgAction(arg);
+                nextArgAction = null;
+                continue;
+            }
+
+            switch (arg)
+            {
+                case "-r":
+                case "--repo":
+                    nextArgAction = value => options.Repo = value;
+                    break;
+                case "-t":
+                case "--tag-name":
+                    nextArgAction = value => options.TagName = value;
+                    break;
+                case "-a":
+                case "--artifacts-folder":
+                    nextArgAction = value => options.ArtifactsFolder = value;
+                    break;
+                default:
+                    ShowUsage();
+                    throw new Exception($"Unknown command line options '{arg}'");
+            }
+        }
+
+        return options;
+    }
+
+    public static void ShowUsage()
+    {
+        Console.WriteLine("Usage: <program> <options>");
+        Console.WriteLine("  -r|--repo              [REQUIRED] Repository (owner/repo)");
```

Comment:
I read this as "owner or repo" and fear others might as well.

Reply:
"Others" is "me in a week".

Reply:
How would you write it to avoid the confusion?

Reply:
```suggestion
        Console.WriteLine("  -r|--repo              [REQUIRED] Repository (as owner/repo)");
```